CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -581,18 +604,30 @@ protected synchronized void updateNotesUI() {
 
             });
         }
-        
+        LOGGER.log(Level.SEVERE, "Size: " + notesToRender.size());
         Platform.runLater(() -> {
-            notesListVBox.getChildren().removeAll(notesListVBox.getChildren());
+            final BooleanProperty foundNoteInEdit = new SimpleBooleanProperty(false);
```

Comment:
for my own curiosity, was there a benefit to using this over the primitive `boolean`?